account/routes.js

```diff
@@ -55,9 +61,14 @@ router.get('/signed-in', async (req, res, next) => {
 
         // Get UserIdToken
         const signInService = createSignInService();
-        const authorisationCode = queryParams.code;
-        const userIdToken = await signInService.getIdToken(authorisationCode);
+        const options = {
```

Comment:
I would be temped to have the signature of `signInService.getIdToken()` to have the `authorisationCode`, and `nonce` as separate parameters, and not combine them in to an object. Same for `tokenService.getIdToken()`, and anywhere else involved. Makes it more readable, and they are not technically "options"